Formula/sdl2_sound.rb

```diff
@@ -0,0 +1,153 @@
+         File extension "MP3"
+         File extension "MP2"
+         File extension "MP1"
+         Written by Ryan C. Gordon <icculus@icculus.org>.
+         https://icculus.org/SDL_sound/
+
+       * Microsoft WAVE audio format
+         File extension "WAV"
+         Written by Ryan C. Gordon <icculus@icculus.org>.
+         https://icculus.org/SDL_sound/
+
+       * Audio Interchange File Format
+         File extension "AIFF"
+         File extension "AIF"
+         Written by TorbjÃ¶rn Andersson <d91tan@Update.UU.SE>.
+         https://icculus.org/SDL_sound/
+
+       * Sun/NeXT audio file format
+         File extension "AU"
+         Written by Mattias EngdegÃ¥rd <f91-men@nada.kth.se>.
+         https://icculus.org/SDL_sound/
+
+       * Ogg Vorbis audio
+         File extension "OGG"
+         Written by Ryan C. Gordon <icculus@icculus.org>.
+         https://icculus.org/SDL_sound/
+
+       * Creative Labs Voice format
+         File extension "VOC"
+         Written by Ryan C. Gordon <icculus@icculus.org>.
+         https://icculus.org/SDL_sound/
+
+       * Raw audio
+         File extension "RAW"
+         Written by Ryan C. Gordon <icculus@icculus.org>.
+         https://icculus.org/SDL_sound/
+
+       * Shorten-compressed audio data
+         File extension "SHN"
+         Written by Ryan C. Gordon <icculus@icculus.org>.
+         https://icculus.org/SDL_sound/
+
+       * Free Lossless Audio Codec
+         File extension "FLAC"
+         File extension "FLA"
+         Written by Ryan C. Gordon <icculus@icculus.org>.
+         https://icculus.org/SDL_sound/
+    EOS
+    if OS.mac?
+      expected += <<~EOS
+
+        \ * Decode audio through Core Audio through
+           File extension "aif"
+           File extension "aiff"
+           File extension "aifc"
+           File extension "wav"
+           File extension "wave"
+           File extension "mp3"
+           File extension "mp4"
+           File extension "m4a"
+           File extension "aac"
+           File extension "caf"
+           File extension "Sd2f"
+           File extension "Sd2"
+           File extension "au"
+           File extension "next"
+           File extension "mp2"
+           File extension "mp1"
+           File extension "ac3"
+           File extension "3gpp"
+           File extension "3gp2"
+           File extension "amrf"
+           File extension "amr"
+           File extension "ima4"
+           File extension "ima"
+           Written by Eric Wing <ewing . public @ playcontrol.net>.
+           https://playcontrol.net
+      EOS
+    end
+    assert_equal expected.strip, shell_output("#{bin}/playsound --decoders").strip
+
+    flags = %W[
+      -I#{include}/SDL2
+      -I#{Formula["sdl2"].include}/SDL2
+      -L#{lib}
+      -L#{Formula["sdl2"].lib}
+      -lSDL2_sound
+      -lSDL2
+    ]
+    Dir["#{pkgshare}/examples/*.c"].each do |r|
+      system ENV.cc, r, "-o", "#{File.basename(r, ".c")}.out", *flags
+    end
```

Comment:
```suggestion
    system ENV.cc, pkgshare/"examples/playsound.c", "-o", "playsound", *flags
    assert_match "help", shell_output("./playsound --help 2>&1", 42)
```

Reply:
@pthariensflame please apply this one.

`playsound_simple` is basically cutted-out version of `playsound` that even don't accept something except sound file and prints nothing except error that there is no file.